easybuild/easyconfigs/a/Abseil/Abseil-20240116.1-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'Abseil'
+version = '20240116.1'
+
+homepage = 'https://abseil.io/'
+description = """Abseil is an open-source collection of C++ library code designed to augment the
+C++ standard library. The Abseil library code is collected from Google's own
+C++ code base, has been extensively tested and used in production, and is the
+same code we depend on in our daily coding lives."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s-cpp/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+checksums = ['3c743204df78366ad2eaf236d6631d83f6bc928d1705dd0000b872e53b73dc6a']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+]
+
+configopts = "-DABSL_PROPAGATE_CXX_STD=ON "
+
+build_shared_libs = True
+
+sanity_check_paths = {
+    'files': ['lib/libabsl_base.so'],
```

Comment:
Please use `SHLIB_EXT` instead of hardcoded `.so`

Reply:
Indeed. I used `--try-update-deps` on PyTorch ;-)

Reply:
I figured as much. I also often forget to change put the `SHLIB_EXT` back after `--try-*` functions :)